gradle/versions.gradle

```diff
@@ -3,9 +3,9 @@ ext {
     // SDK
     minSdkVersion = 16
     automationAppMinSDKVersion = 21
-    targetSdkVersion = 28
-    compileSdkVersion = 28
-    buildToolsVersion = "28.0.3"
+    targetSdkVersion = 29
+    compileSdkVersion = 29
+    buildToolsVersion = "29.0.3"
```

Comment:
This is redundant and ignored by Gradle, you can ditch it.

Reply:
Removed in cacd757ffb514a5c905465340f948058c0c0a8d4